src/TechObject/Mode.cs

```diff
@@ -153,23 +153,7 @@ public string SaveAsLuaTable(string prefix)
                 "\',\n";
 
             res += baseOperation.SaveAsLuaTable(prefix);
-
             string tmp = "";
-            //Совместимость с предыдущей версией (до введения состояния для
-            //операции).
-            if (stepsMngr.Count > 0)
-            {
-                tmp = stepsMngr[0].SaveAsLuaTable(prefix);
-                if (tmp != "")
-                {
-                    res += prefix + "--Совместимость с предыдущей версией" +
-                        " (до введения состояния для операции).\n";
-                    res += tmp;
-                    res += prefix + "--Совместимость с предыдущей версией.\n";
-                    res += "\n";
-                }
-            }
-
             int i = 1;
             string tmp_2 = "";
```

Comment:
Предлагаю тогда и следующий блок кода модернизировать (использовать освободившееся имя переменной `tmp`):
```suggestion
            string tmp = "";
```